docs/learn/faq.md

```diff
@@ -41,13 +41,7 @@ We recommend the following:
    It should work just like over LAN you just have to connect to the public IP itself. Make sure to open UDP on the router. Many forward ports tutorials are available to review, including this option https://portforward.com/.
 1. Use a relay. 
    
-   There is an [unofficial Netcode relay application](https://github.com/MidLevel/MLAPI.Relay) which you can try to host on a server. The relay server uses IP addresses to index hosts. You can connect to it using an IP address and port, but it will be routed through the relay server. You could also use Netcode's list server to browse through all hosts.
-   
-   The other option is to use something like the `SteamP2PTransport`, which will work without the need of setting up any servers if you release your game on Steam.
-
-  :::note About Relays
-  We know there is a high demand for an out-of-the-box relay solution for Netcode, and have plans to resolve in the future.
-  :::
+    [Unity Relay](https://docs.unity.com/relay/introduction.html) offers a way for game developers to securely provide increased connectivity between players by using a join code style workflow without needing to invest in a third-party solution. The Relay service enables developers to connect players together using a listen server. Instead of using dedicated game servers (DGS), the Relay service provides connectivity through a universal Relay server acting as a proxy.
```

Comment:
Could it be worth it to keep the steam mention?

Reply:
If you think so. I removed it since Unity has their own official relay now, so it's a debate on do we keep them on Unity options or provide the Steam alt. Since Relay is still technically beta, I think we could defend either way.
@SamuelBellomo

Reply:
I've seen in our addressables doc "oh we have this CDN you can use, but you can use your own if you want". Our tech is modular, it doesn't cage users in one single ecosystem, to me that's a selling point. But that might be more of a Product question though

Reply:
That's definitely a fair view to see which way we would like to maintain. I'll ask around.

Reply:
@SamuelBellomo I added back the Steam reference for now. We aren't consistent through our documentation on how we handle these types of situations.

Typically in docs realm, we would remove any alt options once a Unity/proprietary option is available. However, with our docs, I feel it's a weird position of being modular, Relay still in beta, and other parts of our documentation having similar references. It may be something we remove in the future when we have a defined stance on this type of references and apply it to all our documentation. Until then, a small reference is fine as long as the preferred method/focus is on the Unity solution.